arch/Mips/MipsMapping.c

```diff
@@ -902,6 +902,9 @@ static name_map group_name_maps[] = {
 	// generic groups
 	{ MIPS_GRP_INVALID, NULL },
 	{ MIPS_GRP_JUMP, "jump" },
+	{ MIPS_GRP_CALL, "call" },
+	{ MIPS_GRP_RET, "ret" },
+	{ MIPS_GRP_INT, "int" },
 
```

Comment:
but because you defined them in mips.h and test_group_name.py, it is good to add them here for consistency.